addons/crm/tests/test_crm_pls.py

```diff
@@ -389,3 +390,44 @@ def test_predictive_lead_scoring(self):
         self.assertEqual(tools.float_compare(lead_tag_1.automated_probability, 28.6, 2), 0)
         self.assertEqual(tools.float_compare(lead_tag_2.automated_probability, 28.6, 2), 0)
         self.assertEqual(tools.float_compare(lead_tag_1_2.automated_probability, 28.6, 2), 0)
+
+        # C. Test Update Probabilities Wizard from the settings
+        # Set the PLS config
+        frequency_fields = self.env['crm.lead.scoring.frequency.field'].search([])
+        pls_fields_str = ','.join(frequency_fields.mapped('field_id.name'))
+        pls_start_date_str = "2000-01-01"
+        IrConfigSudo = self.env['ir.config_parameter'].sudo()
+        IrConfigSudo.set_param("crm.pls_start_date", pls_start_date_str)
+        IrConfigSudo.set_param("crm.pls_fields", pls_fields_str)
+
+        # rebuild frequencies table and recompute automated_probability for all leads.
+        Lead._cron_update_automated_probabilities()
+        leads.invalidate_cache()
+
+        self.assertEqual(tools.float_compare(leads[3].automated_probability, 4.21, 2), 0)
+        self.assertEqual(tools.float_compare(leads[8].automated_probability, 0.23, 2), 0)
+
+        # Check that wizard to update lead probabilities has correct value set by default
+        pls_update_wizard = self.env['crm.lead.pls.update'].create({})
```

Comment:
Could be great for wizards to use the Form tool. That way we ensure it has right values for end users, it is quite easy to use for small wizards.